user/user.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+"fmt"
+"html/template"
+"net/http"
+)
+
+func main(){
```

Comment:
please merge with main to get the latest code, I think we have more endpoints than this